web-server/src/components/Service/SystemStatus.tsx

```diff
@@ -0,0 +1,167 @@
+    [ServiceNames.API_SERVER]: 'Backend Server',
+    [ServiceNames.REDIS]: 'Redis Database',
+    [ServiceNames.POSTGRES]: 'Postgres Database',
+    [ServiceNames.SYNC_SERVER]: 'Sync Server'
+  };
+  return (
+    <FlexBox col gap={2} sx={{ padding: '16px' }}>
+      <Line bold white fontSize="24px" sx={{ mb: 2 }}>
+        System Status
+      </Line>
+      <Divider sx={{ mb: 2, backgroundColor: 'rgba(255, 255, 255, 0.2)' }} />
+
+      {loading ? (
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center'
+          }}
+        >
+          <CircularProgress color="primary" size={50} />
+        </Box>
+      ) : (
+        <FlexBox col gap={2}>
+          {Object.keys(services).map((serviceName) => {
+            const ServiceName = serviceName as ServiceNames;
+            const { isUp } = services[ServiceName];
+            return (
+              <CardRoot
+                key={serviceName}
+                onClick={() => {
+                  dispatch(serviceSlice.actions.setActiveService(serviceName));
+                  dispatch(serviceSlice.actions.setLoading(true));
+
+                  addPage({
+                    page: {
+                      ui: 'system_logs',
+                      title: `${ServiceTitle[serviceName]} Logs`
+                    }
+                  });
+                }}
+                sx={{
+                  transition: 'box-shadow 0.2s ease',
+                  '&:hover': {
+                    boxShadow: '0 4px 15px rgba(0, 0, 0, 0.1)'
+                  },
+                  backgroundColor: 'rgba(255, 255, 255, 0.05)',
+                  borderRadius: '12px',
+                  border: `1px solid ${
+                    isUp ? 'rgba(0, 255, 0, 0.3)' : 'rgba(255, 0, 0, 0.3)'
+                  }`,
+                  padding: '16px',
+                  cursor: 'pointer',
+                  position: 'relative',
+                  overflow: 'hidden'
+                }}
+              >
+                <FlexBox col flexGrow={1} minHeight="5em">
+                  <FlexBox alignCenter justifyContent="space-between">
+                    <Line
+                      white
+                      bold
+                      sx={{
+                        fontSize: '1.2em',
+                        display: 'flex',
+                        alignItems: 'center'
+                      }}
+                    >
+                      {ServiceTitle[serviceName]}
+                      <Box
+                        component="span"
+                        sx={{
+                          display: 'inline-block',
+                          marginLeft: '6px',
+                          width: '10px',
+                          height: '10px',
+                          borderRadius: '50%',
+                          backgroundColor: isUp ? '#28a745' : '#dc3545'
+                        }}
+                      ></Box>
```

Comment:
[optional]:
if empty component, `<Box .... />`

Reply:
got it